### PR TITLE
feat: add news related live views

### PR DESCRIPTION
**Pull request by ruilopesm**

### CODE REVIEW

**Comment by ruilopesm** [comment]
I think "News" is not a good name, for two main reasons:
1. The plural and singular are the same
2. It makes hard to read the code for the above reason

What do you guys (@cesium/atomic-23-24) think? Does anyone have a better name suggestion?

File: priv/repo/seeds/news.exs [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 
   def run do
     seed_news()
   end
 
   def seed_news() do
-    case Repo.all(New) do
+    case Repo.all(News) do
       [] ->
         organizations = Repo.all(Organization)
```
Comment (danielsp45) [inline priv/repo/seeds/news.exs:13]: Don't forget to change `%New{}` to `%News{}` in the for loop.
The seeds are failing.

**Review by danielsp45 — CHANGES_REQUESTED** [review]

**Comment by danielsp45** [comment]
image
The Edit and Delete buttons are looking a bit inconsistent in terms of positioning

**Comment by ruilopesm** [comment]
@danielsp45 Can you check if the positioning is better now? 🙏🏼

**Review by danielsp45 — APPROVED** [review]
Great work 🚀

**Pull request merged**